commands/choose.test.js

```diff
@@ -0,0 +1,19 @@
+const choose = require('./choose.js');
+const messageStub = require('../stub/messageStub');
+
+
+test('Returns the correct choice with the spaces preserved', async () => {
+	// Arrange
+	const message = new messageStub();
+	const mockMath = Object.create(global.Math);
+	mockMath.random = () => 0;
+
+	global.Math = mockMath;
+
+	// Act
+	const responseMessage = await choose.execute(null, message, ['Knights', 'of', 'Academia,', 'Academia', 'of', 'Knights']);
+
+	// Assert
+	const expectedResponse = 'Horace says... Knights of Academia!';
+	expect(responseMessage).toBe(expectedResponse);
+});
```

Comment:
Does test the full functionality of the command? Can it ever pick the second or possibly object? What about using choose with no args?